netbare-sample/src/main/kotlin/com/github/megatronking/netbare/sample/MainActivity.kt

```diff
@@ -34,9 +49,44 @@ class MainActivity : AppCompatActivity(), NetBareListener {
             if (mNetBare.isActive) {
                 mNetBare.stop()
             } else{
+                // TO-DO: Add a check to make sure both certs have been provided
```

Comment:
Is this TODO planned to be added in a later version?

Reply:
I just changed the main button logic: when either both certificates are provided or "use random certificates" is selected button is enabled, disabled otherwise...